indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -519,3 +519,43 @@ th.i-table {
         }
     }
 }
+
+#track-list {
+    list-style-type: none;
+    padding-left: 0;
+    width: 800px;
+
+    .i-box {
```

Comment:
When we have a track without a description we want to show only the i-box header with the title and nothing in the i-box-content , but on default there is 10px of white space after the header so all of that is to get rid of it and also when we do have a description so it looks nice.